### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [x] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Review by Vorobeyko — APPROVED** [review]

File: lib/command/init.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const defaultConfig = {
   mocha: {},
 };
 
-const helpers = ['WebDriver', 'Puppeteer', 'TestCafe', 'Protractor', 'Nightmare', 'Appium', 'Playwright'];
+const helpers = require('../plugin/standardActingHelpers').slice().filter(e => e !== 'Mochawesome');
```
Comment (DavertMik) [inline lib/command/init.js:24]: Mochawesome is not **acting** helper. Why it is in this list?

Maybe we could have differnet lists of helpers and pick the right one (or merge differnet lists) depending on context?

**Review by DavertMik — CHANGES_REQUESTED** [review]
Also i am not sure we need code polish for Master branch as this may harden merging with 3.0. Could you check that this changes can be rebased (merged) to 3.0 branch?

File: lib/command/init.js (hunk 1 above)
Comment (PeterNgTr) [reply lib/command/init.js:24]: @DavertMik I'm not really sure why it exists in the `standardActingHelpers` list, would we remove it?

**Comment by PeterNgTr** [comment]
> Also i am not sure we need code polish for Master branch as this may harden merging with 3.0. Could you check that this changes can be rebased (merged) to 3.0 branch?

not a problem, let me try to rebase with v3.0

**Pull request merged**